stock_picking_actual_date/models/stock_move.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023-2024 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import pytz
+
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    actual_date = fields.Date(
+        compute="_compute_actual_date",
+        store=True,
+    )
```

Comment:
Can we also have the corresponding field in stock.move.line?  In many situations, users also see records in the moves history list view.